test/test_other.py

```diff
@@ -7673,6 +7673,12 @@ def test_umask_0(self):
     self.run_process([EMCC, 'src.c'])
     self.assertContained('hello, world!', self.run_js('a.out.js'))
 
+  @crossplatform
+  @also_with_wasmfs
+  def test_umask(self):
+    self.set_setting("FORCE_FILESYSTEM")
+    self.do_runf('stat/test_umask.c', 'success')
```

Comment:
Can you move this test into `test/other`?   And then use `do_other_test` to run it?   You can run it with `--rebase` to generate the expected output.

Reply:
Oh, sorry, I forgot to relocate the file after https://github.com/emscripten-core/emscripten/pull/22589/commits/b0d7dcf9c6d0e9af51a15a80a6c1ccea6ce759f4.

Reply:
I've addressed this in https://github.com/emscripten-core/emscripten/pull/22589/commits/9619d38a04e111cfa7b9d49187101f431f241fa5.